src/app/shared/components/config-form-group-array/config-form-group-array.component.scss

```diff
@@ -25,3 +25,38 @@ div .form-group-form-fields {
         margin-right: 20px;
     }
 }
+
+@import "variables.scss";
```

Comment:
Move `import` at the top of file, is more understandable

Reply:
Fixed